pkg/cloudprovider/cloudprovider.go

```diff
@@ -90,7 +88,7 @@ func New(instanceTypeProvider *instancetype.Provider, instanceProvider *instance
 	}
 }
 
-// Create a machine given the constraints.
+// Create a nodeClaim given the constraints.
```

Comment:
```suggestion
// Create a NodeClaim given the constraints.
```

Figure that we want to capitalize as a standard here